start_cerberus.py

```diff
@@ -137,7 +138,12 @@ def main(cfg):
             inspect.delete_inspect_directory()
 
         # get list of all master nodes to verify scheduling
-        master_nodes = kubecli.list_nodes("node-role.kubernetes.io/master")
+        master_nodes = []
+        if watch_schedulable_masters:
+            master_nodes = kubecli.list_nodes("node-role.kubernetes.io/master")
+            if len(master_nodes) == 0:
+                logging.info("Unable to list the nodes with label node-role.kubernetes.io/master,\
+                     try setting watch_schedulable_masters to false in the config")
```

Comment:
Maybe exit(1) for the user to make the config change? Thoughts?

Reply:
Yes, it would be better to exit.

Updated :+1: